fcrepo-persistence-ocfl/src/test/java/org/fcrepo/persistence/ocfl/impl/DbFedoraToOcflObjectIndexTest.java

```diff
@@ -175,6 +176,23 @@ public void testRollback() throws Exception {
         }
     }
 
+    @Test
+    public void identifierTooLong() throws InvalidResourceIdentifierException {
```

Comment:
This is a good test, and we should keep it. However, it only tests against H2:
https://github.com/fcrepo/fcrepo/blob/30a516136e8b1009d9266ae795e17a581ba2fd0a/fcrepo-persistence-ocfl/src/test/java/org/fcrepo/persistence/ocfl/impl/DbFedoraToOcflObjectIndexTest.java#L53-L54

If we add a similar integration test to FedoraLdpIT, the GitHub Actions builds will test it against all three supported databases. You should be able to add a test in here:
https://github.com/fcrepo/fcrepo/blob/main/fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

..and you will be good to go.
Sorry for the lack of specificity in my previous comment. The integration tests (those that end in **IT.java) run against a deployed Fedora application... and GitHub Actions are configured to run the tests three times.